Add information about failed Keycloak start

### Description

When the Keycloak startup fails, the only log provided is the following: 

```
Starting Keycloak.................................................. timeout!
Error: Process completed with exit code 1.
```

This is not helpful to find the cause. Therefore it should output the log of the Keycloak server that might show exceptions or other information.

### Discussion

main

### Motivation

Provide more information to analyze build failures.

### Details

_No response_

--- a/scripts/start-local-server.sh
+++ b/scripts/start-local-server.sh
@@ -18,6 +18,7 @@ function waitForServer {
       sleep 1
     else
       echo " timeout!"
+      cat keycloak.log
       exit 1
     fi
   done

